### PR TITLE
[fix] Print warnings to stderr

### PR DESCRIPTION
**Pull request by arturmelanchyk**

### CODE REVIEW

**Comment by qrdl** [comment]
Is this change related to any open issue? If yes, please specify it in description using `Closes #<issue>`

**Comment by tomekkolo** [comment]
> Is this change related to any open issue? If yes, please specify it in description using `Closes #<issue>`

no, it is not :) improvement.

**Review by tomekkolo — APPROVED** [review]

File: pkg/cmd/verify/bom.go [hunk 1]
```diff
@@ -143,7 +144,7 @@ func processBOM(lcUser *api.LcUser, signerID, output, hash, path string) (artifa
 	lowestLevel := artifact.Trusted
 	for i := range deps { // Authenticate mutates the dependency, so use the index
 		if errs[i] != nil {
-			fmt.Printf("cannot authenticate %s@%s dependency: %v\n",
+			fmt.Fprintf(os.Stderr, "cannot authenticate %s@%s dependency: %v\n",
 				deps[i].SignerID, deps[i].Version, errs[i])
```
Comment (qrdl) [inline pkg/cmd/verify/bom.go:148]: It should be `deps[i].Name` instead of `deps[i].SignerID`

Comment (arturmelanchyk) [reply pkg/cmd/verify/bom.go]: fixed

**Review by qrdl — APPROVED** [review]

**Pull request merged**